llm_studio/python_configs/text_causal_language_modeling_config.py

```diff
@@ -20,6 +20,78 @@
 from llm_studio.src.utils.modeling_utils import generate_experiment_name
 
 
+@dataclass
+class ConfigNLPCausalLMDataset(DefaultConfig):
+    dataset_class: Any = (
+        llm_studio.src.datasets.text_causal_language_modeling_ds.CustomDataset
+    )
+    train_dataframe: str = "/path/to/train.csv"
+    validation_strategy: str = "automatic"
+    validation_dataframe: str = ""
+    validation_size: float = 0.01
+
+    data_sample: float = 1.0
+    data_sample_choice: Tuple[str, ...] = ("Train", "Validation")
+
+    prompt_column: Tuple[str, ...] = ("instruction", "input")
+    answer_column: str = "output"
+    parent_column: str = "None"
```

Comment:
Maybe rename this to `parent_id_column` ? The default dataset also uses `parent_id` as a column name